src/Heading/__tests__/Heading.test.tsx

```diff
@@ -5,7 +5,7 @@ import Heading from '../Heading';
 (['H1', 'H2', 'H3', 'H4', 'H5', 'H6'] as const).forEach((variant) => {
   test(`renders a ${variant}`, () => {
     const Variant = Heading[variant];
-    const tree = createWithTheme(<Variant>Hello World</Variant>);
+    const tree = createWithTheme(<Variant color="white">Hello World</Variant>);
```

Comment:
Updated the test to exercise the `color` prop.  This wouldn't have compiled before the change.